.github/workflows/integration_tests.yml

```diff
@@ -591,7 +585,7 @@ jobs:
 
           test_outcomes=(
             "${{ steps.keptn_install.outcome }}"
-            "${{ steps.authenticate_keptn_cli.outcome }}"
+            "${{ steps.determine_keptn_endpoint.outcome }}"
```

Comment:
Does it have any special value to have the `determine_keptn_endpoint` in the integration tests report on GH?
If no, you could remove it here

Reply:
good catch, I think does does not provide any meaningful value. I have removed it